app/views/admin/assets/show.html.erb

```diff
@@ -1,20 +1,16 @@
-<div>Managing an Asset</div>
+<div>Managing an Asset <%= "(this asset is a collection thumbnail)" if @asset.collection_thumbnail? %></div>
 <h1><%= @asset.title %></h1>
 <%if @asset&.parent %>
-  <% if @asset.parent.is_a? Work %>
-    <p>In work: <%= link_to @asset.parent.title, [:admin, @asset.parent] %> </p>
-  <% else %>
-    <p>Thumbnail for collection: <%= link_to @asset.parent.title, collection_path(@asset.parent) %> </p>
-  <% end %>
-
+  <p>Asset is a member of <%= link_to @asset.parent.title, parent_path(@asset) %> </p>
```

Comment:
Hmm, so now we only can tell if this is a collection thumbnail or not by recognizing the title, or cicking on it to see where `parent_path` takes you?

I think I prefer the former way this was very explicitly labelling this as a collection thumb here.  I appreciate simplfying/DRYing the logic as you are doing in this PR, but can we leave the UI really distinguishing with explicit labels what's going on?